versions.json

```diff
@@ -1,5 +1,34 @@
 {
   "versions": [
+    {
+      "path": "4.0.0-preview1/scala2.13-java21-python3-ubuntu",
+      "tags": [
+        "4.0.0-preview1-scala2.13-java21-python3-ubuntu",
+        "4.0.0-preview1-java21-python3",
+        "4.0.0-java21",
+        "python3-java21"
```

Comment:
I'm okay with this, anther choice maybe better to add this after 4.0.0 published?

Reply:
Let me remove this. :)